ethers-solc/src/artifacts/mod.rs

```diff
@@ -837,6 +841,112 @@ pub struct MetadataSource {
     pub license: Option<String>,
 }
 
+/// Model checker settings for solc
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct ModelCheckerSettings {
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub contracts: Option<BTreeMap<String, Vec<String>>>,
```

Comment:
```suggestion
    #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
    pub contracts: BTreeMap<String, Vec<String>>,
```

Reply:
Given this suggestion, would it also make sense to do the same for `    pub targets: Option<Vec<ModelCheckerTarget>>,`?

Reply:
Also this forces the test to be changed to
```
    project.project_mut().solc_config.settings.model_checker = Some(ModelCheckerSettings {
        contracts: BTreeMap::new(),
        ...
```
Is there a reason why not using `Option` is preferred?